rand_distr/src/lib.rs

```diff
@@ -77,6 +77,7 @@
 //! - Misc. distributions
 //!   - [`InverseGaussian`] distribution
 //!   - [`NormalInverseGaussian`] distribution
+//!   - [`Gumbel`] distribution
```

Comment:
Maybe this should go to the same category as `Weibull`?

Reply:
I think that's fair, since it's closely related to the exponential as well which is also in that group